src/ui/content/movies.tsx

```diff
@@ -14,40 +14,44 @@ interface Movie {
     title: string
 }
 
+interface lmProps {
+  searchString: string
+}
+
+function ListMovies(props: lmProps) {
+  const search = props.searchString
+  
+  const { loading:isLoading, error, data } = useQuery(MOV_Q, { variables: { search: search } })
+    
+  if (isLoading) return <p>Loading ...</p>
+  if (error) return <h1>Log in to view movies.</h1>
+
+  if (data.movies.length === 0) return <h1>No movies found.</h1>
+  const movies = data.movies
+  const listItems = movies.map((movie: Movie, index: number) =>
+        <h1 key={index}>{movie.title}</h1>
```

Comment:
using an index as key is rarely a good idea, it might have confusing side effects. Best solution is to add the movie id to the response and use that:
```suggestion
        <h1 key={movie.id}>{movie.title}</h1>
```